### PR TITLE
Extended profile - story tab

### PR DESCRIPTION
**Pull request by mmmavis**
Related to #929 

https://github.com/mozilla/network-pulse/issues/929#issuecomment-395879668

- [x] show a "Story" tab if the profile has user_bio_long property in backend
- [x] create a component that renders markdown -> HTML since we are storing user_bio_long as **markdown** in the backend

We might wanna tweak the text stylings used to render user's long bio. Currently the long bio section just inherit existing CSS like

screen shot 2018-06-20 at 9 24 26 am

### CODE REVIEW

**Comment by mmmavis** [comment]
@xmatthewx can you help do design review?

**Comment by mmmavis** [comment]
@xmatthewx PR updated!

File: pages/profile.jsx [hunk 1]
```diff
@@ -85,13 +85,22 @@ class Profile extends React.Component {
...
-      <hr />
-      <ProfileTabGroup
-        profileId={userProfile.profile_id}
-        myProfile={userProfile.my_profile}
-        entryCount={userProfile.entry_count}
-        activeTab={activeTab}
-      />
+      <div className="row">
+        <div className="col-12">
+          <hr />
```
Comment (gvn) [inline pages/profile.jsx]: Does this `hr` really need it's own row? Could it just go right before line 95?

**Comment by xmatthewx** [comment]
Excellent! @mmmavis ping me when this is on staging. I'll will play around with the markdown and see if we need a follow up ticket for styles.

File: pages/profile.jsx (hunk 1 above)
Comment (mmmavis) [reply pages/profile.jsx]: good point. will update!

**Comment by mmmavis** [comment]
@xmatthewx this is ready for review on staging https://network-pulse-staging.herokuapp.com/profile/8

**Comment by xmatthewx** [comment]
Were you able to set your multi-line via django? My linebreaks were not preserved. Hmmm. 

Either way, I think this is good to ship to production next week. We can just continue to poke at markdown in that new ticket.

**Comment by mmmavis** [comment]
@xmatthewx i had to use `\n` to start a new line. e.g.,

```
# (h1) title \n ## (h2) header 2 \n hello hello hello
```

Can we change the Django admin so that field is a `textarea`(support line break) instead of single-line text input box? (let's discuss this in a new ticket instead)

**Comment by xmatthewx** [comment]
Tracking API discussion here: https://github.com/mozilla/network-pulse-api/issues/381

**Pull request merged**